language-ruby_5bb9e082042863158cc7236d.md

```diff
@@ -92,6 +93,34 @@ Semaphore doesn't set language specific environment variables like
             - bundle exec rake test
 </code></pre>
 
+## Rails
+
+Configuring Rails applications require a few extra steps. You'll need
```

Comment:
This sentence can be removed without effect, so let's remove it. We can assume that many readers will jump straight to this section. And always prefer opening in a positive tone, "xyz requires extra steps" sounds like ok this isn't supported but here's a workaround.